extensions/wikia/Piggyback/Piggyback_body.php

```diff
@@ -143,21 +139,22 @@ function validPiggyback() {
 		global $wgUser;
 		/* pre valid */
 		$cUserId = User::idFromName( $this->mUsername );
-		if( $this->mUsername != "" && $wgUser->getID() != $cUserId ) {
-			$this->mainLoginForm( wfMsg( 'piggyback-wronguser', htmlspecialchars( $this->mUsername ) ) );
-			return ;
-		}
+		if ( $this->mUsername != "" && $wgUser->getID() != $cUserId ) {
```

Comment:
Would it make sense to use a wrapping if($this->mUsername != "") statement to check it only once (this would wrap if statements from this line and line 148 :v: